alpha/scripts/utils/setPartnerAdditionalParameters.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+chdir(dirname(__FILE__));
+require_once(dirname(__FILE__) . '/../bootstrap.php');
+
+if (count($argv) !== 3)
+{
+        die('please provide a partner id and a mailing list ' . PHP_EOL .
```

Comment:
I think it would be clearer if you just wrote something like:
Usage: basename(__FILE__) <partner_id> <mailing-list>